platforms/hdp2.6.1/docs/HDP-installation.md

```diff
@@ -74,31 +75,47 @@ After install the rpm,restart ambari-server using 'service ambari-server restart
 Follow the steps to install splicemachine server.
 
-1.click the action button on the left buttom of the ambari page,then click on 'Add Services'
+1. Click the action button on the left bottom of the ambari page,then click on 'Add Services'
 
 <img src="docs/add_services.jpg" alt="Add Service" width="400" height="200">
 
-2.choose splice machine from the 'add service wizard'
+2. Choose splice machine from the 'add service wizard'
 
 <img src="docs/add_service_wizard.jpg" alt="Add Service Wizard" width="400" height="200">
 
-3.choose hosts needed to install splice machine,only choose hosts that have hbase region server installed.Then click next.
+3. Choose the master machine. It need to be the same machine with HBase master.
+
+4. Choose hosts needed to install splice machine,only choose hosts that have hbase region server 
+installed.Then click next.
 
 <img src="docs/choose_hosts.jpeg" alt="Choose hosts" width="400" height="200">
 
-4.O On the page of custom services,no properties need to customize by hand.
+5. On the page of custom services,no properties need to customize by hand unless you would like to
+ add Apache Ranger Support.
 
 <img src="docs/custom_services.jpeg" alt="Custom Services" width="400" height="200">
 
-5.please review all the configuration change made by ambari and click OK to continue.
+6. Please review all the configuration change made by ambari and click OK to continue.
 
 <img src="docs/dependent_config.jpeg" alt="dependent_config.jpeg" width="400" height="200">
 
-6.please click next all the way down to this page ,then click 'deploy',after that finishes,splice machine is installed.
+**Note**: Ambari will not show all the recommend values in some situations. Make sure these important configurations are set properly by click "recommend" button next to the configs:
+
+(1) In HBase's config "Advanced hbase-site", make sure `hbase.coprocessor.master.classes` includes `com.splicemachine.hbase.SpliceMasterObserver`.
+
+(2) In HBase's config "Advanced hbase-site", make sure `hbase.coprocessor.regionserver.classes` includes `com.splicemachine.hbase.RegionServerLifecycleObserver`.
+
+(3) In HBase's config "Advanced hbase-site", make sure `hbase.coprocessor.region.classes` includes `org.apache.hadoop.hbase.security.access.SecureBulkLoadEndpoint,com.splicemachine.hbase.MemstoreAwareObserver,com.splicemachine.derby.hbase.SpliceIndexObserver,com.splicemachine.derby.hbase.SpliceIndexEndpoint,com.splicemachine.hbase.RegionSizeEndpoint,com.splicemachine.si.data.hbase.coprocessor.TxnLifecycleEndpoint,com.splicemachine.si.data.hbase.coprocessor.SIObserver,com.splicemachine.hbase.BackupEndpointObserver`. If the property is not found, you can add the property in "Custom hbase-site".
+
+(4) In Hbase's config "hbase-env template", make sure the comments like "Splice Specific Informatio" are in the configurations.
+
+
+7. Please click next all the way down to this page ,then click 'deploy',after that finishes,splice
+ machine is installed.
 
```

Comment:
Punctuation.  Change to "7. Please click next all the way down to this page, then click 'deploy'. After that finishes, Splice Machine is installed.

Also, may want to consider capitalizing all occurrences of "splice machine".